scss/_debug.scss

```diff
@@ -0,0 +1,44 @@
+@if $enable-debugger-classes {
+  .debug {
+    &::before {
+      position: fixed;
+      top: 0;
+      left: 0;
+      z-index: 2000;
+      padding: $spacer * .5 $spacer;
+      font-family: $font-family-base;
+      color: $black;
+      background-color: $white;
+      border-radius: 0 0 4px; // stylelint-disable-line property-disallowed-list
+      @include font-size($h1-font-size);
+    }
+
+    @each $breakpoint, $value in $grid-breakpoints {
+      @include media-breakpoint-up($breakpoint) {
+        &::before {
+          content: "#{$breakpoint}";
+        }
+      }
+    }
+
+    [class^="container"],
```

Comment:
Can you provide this functionality as a seperated class name?

Reply:
Why not, I don't strong opinion on this. I did it this way because on my project, we use sometimes `.row` and sometimes `.container-*` with `.col-*` inside. Do you have any cool name to propose ?

Reply:
This feature looks nice but the developer may want to toggle on/off this option to make the design look more clear and understandable.

You know, design considerations call for pixel-perfect work, and the overlay of this option can become annoying at some point.

`.bs-debug-grid-marks`, `.bs-debug-grid-cols` or something else like to understand of the purpose this utility. 

It would be better if it does not have any connection with the other class name. The developer can use whatever he/she wants like `<body class="bs-debug-grid-breakpoints bs-debug-grid-marks">`